components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-service-provider.jsp

```diff
@@ -1390,7 +1489,146 @@
+                            </tr>
+                            <%}%>
+                        </table>
+                        <table style="padding-top: 5px; padding-bottom: 10px;" class="carbonFormTable">
+                            <tr>
+                                <td class="leftCol-med labelField"><a id="saml_link" class="icon-link" 
+                                    onclick="onAppPurposesManageClick()"><fmt:message key="manage.app.consent.purposes"/></a></td>
+                            </tr>
+                        </table>
+                        <table style="padding-top: 15px; padding-bottom: 10px;" class="carbonFormTable" id="general_purposes_combo">
+                                <tr>
+                                    <td class="leftCol-med labelField"><fmt:message key="general.consent.purposes"/></td>
+                                </tr>
+                                <tr>
+                                    <td class="leftCol-med">
+                                            <select class="leftCol-med" id="general_purposes" name="general_purposes"
+                                            style=" margin-left: 5px; ">
+                                                <option value="not_selected">---Select---</option>
+                                                <%for (Purpose generalPurpose : generalPurposes) {%>
+                                                <option
+                                                    value="<%="row_general_purpose_id_" + generalPurpose.getId()%>"><%=Encode.forHtmlContent(generalPurpose.getName())%>
+                                                </option>
+                                                <%}%>
+                                            </select>
+                                            <input id="addGeneralConsentBtn" type="button" value="Add" onclick="onGeneralPurposesClick()"/>
+                                    </td>
+                                </tr>
+                        </table>
+                        <table class="styledLeft" id="general_purposes_tbl">
+                            <thead>
+                                <tr>
+                                    <th class="leftCol-big"><fmt:message key="purpose"/></th>
+                                    <th class="leftCol-big"><fmt:message key="description"/></th>
+                                    <th class="leftCol-mid"><fmt:message key="display.order"/></th>                                    
+                                    <th class="leftCol-mid"><fmt:message key="remove"/></th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <%if (appGeneralPurposes == null || appGeneralPurposes.isEmpty()) {%>
+                                <script>
+                                    $(jQuery('#general_purposes_tbl')).hide();
```

Comment:
Use either 
`$('#general_purposes_tbl').hide();`
or
`jQuery('#general_purposes_tbl').hide();`

Both methods achieve the same, no need to wrap a jQuery object again with jQuery